src/test/rustdoc-ui/coverage/json.stdout

```diff
@@ -0,0 +1 @@
+{"$DIR/json.rs":{"total":13,"with_docs":7}}
```

Comment:
This test doesn't yet pass on Windows because of how `$DIR` is normalized. Specifically https://github.com/rust-lang/rust/blob/abc3073c92df034636a823c5382ece2186d22b9e/src/tools/compiletest/src/runtest.rs#L3213-L3215 isn't being triggered but is needed in this case. This can be fixed by adding `cflags.contains("--output-format json") || cflags.contains("--output-format=json")` to https://github.com/rust-lang/rust/blob/abc3073c92df034636a823c5382ece2186d22b9e/src/tools/compiletest/src/runtest.rs#L3204-L3207.

Reply:
Great catch, thanks a lot!